packages/emile/emile.0.7/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name:         "emile"
```

Comment:
This is a side-note but the name field is unnecessary here (and probably in the upstream repository as well)